qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -107,6 +107,10 @@
                     fs_dest_fs3 = '/mnt/${fs_target_fs3}'
                     fs_dest_fs4 = '/mnt/${fs_target_fs4}'
                     fs_dest_fs5 = '/mnt/${fs_target_fs5}'
+                - with_reboot:
+                    only default..with_cache.auto.default
+                    reboot_guest = True
```

Comment:
By the way, there is no need to write `True` here, I think it's better to write `yes`.

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>